spec/services/assessors/main_assessor_spec.rb

```diff
@@ -2,117 +2,15 @@
-        expect(%w[ineligible pending eligible]).to have_main_assessment_result(assessment, 'ineligible')
-        expect(%w[ineligible pending ineligible]).to have_main_assessment_result(assessment, 'ineligible')
-        expect(%w[ineligible pending pending]).to have_main_assessment_result(assessment, 'ineligible')
-        expect(%w[pending contribution_required contribution_required]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending contribution_required eligible]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending contribution_required ineligible]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending contribution_required pending]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending eligible contribution_required]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending eligible eligible]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending eligible ineligible]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending eligible pending]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending ineligible contribution_required]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending ineligible eligible]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending ineligible ineligible]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending ineligible pending]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending pending contribution_required]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending pending eligible]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending pending ineligible]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-        expect(%w[pending pending pending]).to have_assessment_error(assessment, 'Assessment not complete: Gross Income assessment still pending')
-      end
-    end
+    # TODO: write tests here that look at all the assessment_eligibility results and resturn eligible, ineligible, partly eligible
```

Comment:
Should these have been written or is it a longer term aim?

Reply:
Oh no, I still need to write thse tests I think.